src/test/kotlin/uk/gov/justice/digital/hmpps/nomisprisonerapi/helper/builders/PersonAddressBuilder.kt

```diff
@@ -0,0 +1,31 @@
+package uk.gov.justice.digital.hmpps.nomisprisonerapi.helper.builders
+
+import uk.gov.justice.digital.hmpps.nomisprisonerapi.jpa.AddressPhone
+import uk.gov.justice.digital.hmpps.nomisprisonerapi.jpa.Person
+import uk.gov.justice.digital.hmpps.nomisprisonerapi.jpa.PersonAddress
+
+class PersonAddressBuilder(
+  var premise: String = "41",
+  var street: String = "High Street",
+  var locality: String = "Sheffield",
+  private var phoneNumbers: List<Pair<String, String>> = listOf()
```

Comment:
no bothered to do much more than these 3, a future version could set city and country if required.